ui/src/choice_group.tsx

```diff
@@ -69,13 +71,25 @@ export const
       onChange = (_e?: React.FormEvent<HTMLElement>, option?: Fluent.IChoiceGroupOption) => {
         if (option) wave.args[m.name] = option.key
         if (m.trigger) wave.push()
+      },
+      styles: Fluent.IChoiceGroupStyles = {
+        flexContainer: {
+          display: 'flex',
+          flexWrap: 'wrap',
+          selectors: {
+            '.ms-ChoiceField': {
+              marginRight: 15,
+            }
```

Comment:
For this one you need to update styles of each particular option (L70) to ` styles: { choiceFieldWrapper: { marginRight: 15 } }`

Reply:
done